website/styles/Theme/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  type ThemeConfig,
+  extendTheme,
+  usePrefersReducedMotion,
+} from "@chakra-ui/react";
+import { containerTheme } from "./Components/Container";
+import { StyleFunctionProps, Styles } from "@chakra-ui/theme-tools";
+
+const config: ThemeConfig = {
+  initialColorMode: "system",
+  useSystemColorMode: false,
+  disableTransitionOnChange: true,
+};
+
+const components = {
+  Container: containerTheme,
+  Box: (props: StyleFunctionProps) => ({
```

Comment:
Can you break these theme configs into their on files that are next to `Container.ts`?

Reply:
forgot to do this. Will work on it and push in a separate pr if that's okay

Reply:
Sure